### ISSUE TITLE
Consider using callbacks for updating SkeletonMarker-s in WorldInteractiveMarkerViewer

### ISSUE THREAD

**Issue by jslee02** [personalrobotics/aikido#254]
https://github.com/personalrobotics/aikido/pull/252#issue-269541118

> At some point, we could fix this by registering callbacks to World that are called when adding or removing Skeletons.

We could use [`dart::common::Signal<...>`](https://github.com/dartsim/dart/blob/37bec25fea9d67f7388a67b54d7b48fb47153df7/dart/common/Signal.hpp#L106) for this.

**Comment by psigen** [comment]
Agreed, this would be the preferred mechanism, and basically exactly what
DART signals are intended for.
